packages/datadog-plugin-fastify/test/integration-test/helper.mjs

```diff
@@ -0,0 +1,18 @@
+export async function createAndStartServer (app) {
+  app.get('/', async (request, reply) => {
+    return 'hello, world\n'
+  })
+
+  const startServer = async () => {
+    try {
+      await app.listen(0)
+      const address = app.server.address()
+      const port = address.port
+      process.send({ port })
+    } catch (err) {
+      process.exit(1)
+    }
+  }
+  startServer()
+  return startServer
```

Comment:
Why have that nested startServer function? It just calls it immediately and then returns the function rather than the result for some reason even though it never seems to get used. 🤔 

Could simplify to this if you're never actually going to _need_ that nested function.

```suggestion
  try {
    await app.listen(0)
    const address = app.server.address()
    const port = address.port
    process.send({ port })
  } catch (err) {
    process.exit(1)
  }
```